backend/app/api/ws/chat_namespace.py

```
@@ -78,6 +78,80 @@
+    )
+    task_json = task.json if task else {}
+    team_ref = ((task_json or {}).get("spec") or {}).get("teamRef") or {}
+    team_name = team_ref.get("name")
+    if not team_name:
+        return None
+
+    query = db.query(Kind.id).filter(
+        Kind.kind == "Team",
+        Kind.name == team_name,
+        Kind.namespace == team_ref.get("namespace", "default"),
+        Kind.is_active == True,
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)